README.md

```diff
@@ -97,61 +87,74 @@ To disable, run:
 invoke remove_hooks
 ```
 
-### Creating a new test database
-To run the API locally you will need a database, we provide a sql dump that has a small sample of data to get you running locally.
+#### Create local databases
+Before you can run this project locally, you'll need an API database and a test database. 
 
-To create your own test database, make sure you have postgres and run:
+To create your databases, run:
 ```
 createdb cfdm_test
+createdb cfdm_unit_test
+```
+Load our sample data into the development database (`cfdm_test`) by running:
+```
 pg_restore --dbname cfdm_test data/subset.dump
 ./manage.py update_all
 ```
-(Don't worry if there are some user does not exist error messages)
+Ignore `user does not exist` error messages. Everything will still work!
 
-FEC and 18F members can set the sql connection to one of the RDS boxes with:
+**Note**: *FEC and 18F members can set the SQL connection to one of the RDS boxes with:*
 ```
-export SQLACONN=<psql:address-to-box>
+export SQLA_CONN=<psql:address-to-box>
 ```
+*Reach out to a team member to get the actual addresses.* 
 
-### Run locally
-Install your npm packages:
-```
-npm i
-```
+
+#### Run locally
+Follow these steps every time you want to work on this project locally.
 
 Run:
 ```
 ./manage.py runserver
 ```
-The site can be found at [http://localhost:5000](http://localhost:5000)
+View your local version of the site at [http://localhost:5000](http://localhost:5000).
 
 
-We are always trying to improve our documentation, if you have suggestions or run into problems feel free to [file an issue](https://github.com/18F/openFEC/issues).
 
 
-## Deployment
+### Deployment (18F and FEC team members only)
 
-##### Likely only useful for 18F team members
+#### Deployment prerequisites
+If you haven't used Cloud Foundry in other projects, you'll need to install the Cloud Foundry CLI and the autopilot plugin.
 
-Before deploying, install the [Cloud Foundry CLI](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) and the [autopilot plugin](https://github.com/concourse/autopilot):
+1. Read [Cloud Foundry documentation](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) to install Cloud Foundry CLI.
 
+2. Install autopilot by running:
 ```
 cf install-plugin autopilot -r CF-Community
 ```
+[Learn more about autopilot](https://github.com/concourse/autopilot).
+
+#### Deployment steps
+1. [Log in to Cloud Foundry](https://docs.cloudfoundry.org/cf-cli/getting-started.html). Org should be `fec`. Space can be `dev`, `stage`, or `prod`.
```

Comment:
Running `invoke deploy` (step 3) automatically logs in to cloud.gov using the credentials set in step 2. I don't think this step is necessary.